### ISSUE TITLE
Earn 100 in accessibility on Lighthouse audit

### ISSUE THREAD

**Issue by anthony-iacono** [anthony-iacono/oasys#30]
ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
